tests/libnode/node.go

```diff
@@ -458,3 +468,11 @@ func ExecuteCommandOnNodeThroughVirtHandler(nodeName string, command []string) (
 	}
 	return exec.ExecuteCommandOnPodWithResults(virtHandlerPod, components.VirtHandlerName, command)
 }
+
+func ExecuteCommandInVirtHandlerPod(nodeName string, args []string) (stdout string, err error) {
```

Comment:
Do we need this wrapper at all? Looks fairly simple to me.
Also I find the name misleading.

Reply:
It's quite bad to have these two functions (ExecuteCommandOnNodeThroughVirtHandler, ExecuteCommandInVirtHandlerPod), with similar-yet-not-the-same semantics. I'm sure we'd be better off with them merged into one.

I hoped that putting them next to each other would be considered a good-enough progress and a hint for a follow-up PR. But then again, reviewer is king.

Reply:
Hurts to see this, as you said. Merging them now is no option?

Reply:
I've pushed a draft for the merge https://github.com/kubevirt/kubevirt/pull/13104 because I am not sure about my capacity to finish this soon, while I believe that eliminating the thorn-in-the-eye that is io_utils has merits of its own.

Reply:
I think the changes in the PR are already moving us quite a bit forward so we can tackle this in a separate one